schema.json

```
@@ -2,34 +2,16 @@
     "$schema": "http://json-schema.org/draft-07/schema",
     "$id": "https://citation-file-format.github.io/1.2.0/schema",
     "title": "Citation File Format",
-    "description": "A file with citation metadata for software or datasets",
+    "description": "A file with citation metadata for software or datasets.",
     "additionalProperties": false,
     "type": "object",
     "properties": {
-        "cff-version": {
-            "type": "string",
-            "pattern": "^1\\.2\\.0$",
-            "description": "The version of CFF used for providing the citation metadata"
-        },
-        "message": {
-            "type": "string",
-            "description": "A message to the (human) reader of the file to let them know what to do with the citation metadata",
-            "minLength": 1
-        },
-        "type": {
-            "type": "string",
-            "enum": [
-                "dataset",
-                "software"
-            ]
-        },
         "abstract": {
+            "description": "A description of the software or dataset.",
             "type": "string"
         },
         "authors": {
-            "type": "array",
-            "minItems": 1,
-            "uniqueItems": true,
+            "description": "The author(s) of the software",
```

```suggestion
            "description": "The author(s) of the software or dataset.",
```

Add dataset, and full stop.